src/app/forecasting/models/calculate-input.model.ts

```diff
@@ -8,12 +8,13 @@ export class CalculateInput {
   expectedAnnualGrowthRate = 0;
   monthlyContribution = 0;
   leanFiPercentage = 0;
+  public birthdate: Date = null;

```

Comment:
Please remove the `public` to keep it consistent with the existing codebase. By default these variables are exposed to the html files; you explicitly mark it as private if you want to restrict it.

Reply:
Removed access modifier, thanks for the correction!